Resources/doc/2-configuring_resource_owners.md

```diff
@@ -59,6 +59,7 @@ hwi_oauth:
 - [Instagram](resource_owners/instagram.md)
 - [itembase](resource_owners/itembase.md)
 - [Jira](resource_owners/jira.md)
+- [Jawbone](resource_owners/jawbone.md)
```

Comment:
This sounds not related.

Reply:
Yeah, I just forgot to add it, when i was implementing jawbone support